templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -654,6 +696,7 @@ contents:
       ovnk_var_dir='/var/lib/ovnk'
       extra_bridge_file="${ovnk_config_dir}/extra_bridge"
       iface_default_hint_file="${ovnk_var_dir}/iface_default_hint"
+      ip_hint_file="/run/nodeip-configuration/primary-ip"
```

Comment:
It's been a while since I tested UPI, but if `nodeip-configuration` does not run and we are doing this manually we have to set `KUBELET_NODE_IP` and then also create `/run/nodeip-configuration/primary-ip`.

I need to test this, but can ignition just create a file in `/run` or would I need a systemd unit to create the file after `/run` is populated later in boot?

Reply:
nodeip-configuration should run in upi, it is actually same as with SNO and there it is running for sure

Reply:
systemd unit should create it, /run is cleaned on every boot so it will not work with ignition

Reply:
Going forward, if you need consistency between kubelet and configure-ovs you will want to use the NODEIP_HINT interface to nodeip-configuration. That will provide a single interface that ensures every service is using a consistent node ip.

Reply:
Yep, this is sort if your idea :) and this why we are adding it